internal/proto/encryptionkeyrotation_grpc.pb.go

```diff
@@ -0,0 +1,116 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+// versions:
+// - protoc-gen-go-grpc v1.4.0
+// - protoc             v3.20.2
```

Comment:
this is an unrelated change, there should be a commit that updates the protoc version and updated generated files.